test/language/expressions/class/dstr-private-gen-meth-dflt-ary-init-iter-no-close.js

```diff
@@ -0,0 +1,101 @@
+// This file was procedurally generated from the following sources:
+// - src/dstr-binding/ary-init-iter-no-close.case
+// - src/dstr-binding/default/cls-expr-private-gen-meth-dflt.template
+/*---
+description: Iterator is not closed when exhausted by pattern evaluation (private class expression method (default parameter))
+esid: sec-class-definitions-runtime-semantics-evaluation
+features: [Symbol.iterator, class, class-methods-private, generators, destructuring-binding,
+  default-parameters]
```

Comment:
This `features` value is spread across multiple lines, which  https://github.com/tc39/test262/blame/master/tools/packaging/parseTestRecord.py doesn’t seem to handle very well.